server/src/jobs/optout/getAllEtablissements.js

```diff
@@ -0,0 +1,49 @@
+const { Optout } = require("../../common/model");
+const { runScript } = require("../scriptWrapper");
+const axios = require("axios");
+const _ = require("lodash");
+
+const getEtablissements = async (options) => {
+  let { page, allEtablissements } = { page: 1, allEtablissements: [], ...options };
+
+  const response = await axios.get(
+    `https://referentiel.apprentissage.beta.gouv.fr/api/v1/organismes?etat_administratif=actif&qualiopi=true&items_par_page=1000&ordre=desc&page=${page}`
+  );
+
+  const { organismes, pagination } = response.data;
+  allEtablissements = allEtablissements.concat(organismes); // Should be properly exploded, function should be pure
+
+  if (page < pagination.nombre_de_page) {
+    return getEtablissements({ page: page + 1, allEtablissements });
+  } else {
+    return allEtablissements;
+  }
+};
+
+runScript(async ({ etablissement }) => {
+  const referentiel = await getEtablissements();
+  const data = await Optout.find({}).lean();
+
+  const newEtablissement = _.differenceBy(referentiel, data, "siret");
+  const organismes = newEtablissement.filter((x) => x.contacts.length > 0 && x.adresse);
```

Comment:
Use more explicit variables like:

```suggestion
  const organismes = newEtablissement.filter((etablissement => etablissement.contacts.length > 0 && etablissement.adresse);
```